052-cruise-control-api-users.md

```diff
@@ -137,32 +137,61 @@ This is also a pattern used in other Strimzi APIs already, for example Strimzi's
 
 Strimzi would decode and use the contents of this secret to populate the Cruise Control API auth credentials file which would be mounted and used by the Cruise Control pod.
 
-It's worth noting that if a developer were to provide a corrupt configuration file - one which specifies the `ADMIN` role or an incorrect format, the Strimzi Operator would ignore the custom configuration, log a warning, and add a warning condition to the `Kafka` custom resource.
+It's worth noting that if an advanced user were to provide a corrupt configuration file - one which specifies the `ADMIN` role or an incorrect format, the Strimzi Operator would ignore the custom configuration, log a warning, and add a warning condition to the `Kafka` custom resource.
 
-Although this would simplify the overall design it would also shift a lot of the maintanence responsibility to the developer.
-The developer would need to create a valid Cruise Control API user configuration and encode it into a secret.
-That being said, this is still a reasonable amount of responsibility to leave to the developer.
-This is because accessing the Cruise Control API directly is not the primary way of interacting with Cruise Control for a Strimzi-managed Kafka cluster and is expected to be used only by advanced developers in special use-cases.
-However, with proper supporting documentation, this new feature will give developers a greater degree of control and assistance with special use-cases.
+For example, if an advanced user were to provide a corrupt capacity configuration file, a warning condition like the following would be added to the status section of the `Kafka` custom resource :
```

Comment:
What's a capacity configuration file? I would have thought that the warning message in the example should be complaining about a "user config file".

Reply:
Argh, sorry I am mixing up another PR I am working on, it is fixed now!